demos/python/class1.py

```diff
@@ -14,5 +16,4 @@ def method1(this):
                             1,2,3])
 
 c = returnsjson()
-print (c.method1())
-                                  
+print(c.method1())
```

Comment:
Can't tell if there's a newline at the end of this file. Please make sure there is.

Reply:
Yeah, I made sure there was.  It would display the red emoji if there wasn't.

Reply:
Cool thanks - I figured that would be the case but the `- ` on line 18 was odd